.buildkite/e2e/nightly-main-matrix.yaml

```
@@ -3,21 +3,7 @@
   fixed:
     E2E_PROVIDER: gke
   mixed:
-    - E2E_STACK_VERSION: "6.8.23"
     - E2E_STACK_VERSION: "7.17.8"
-    - E2E_STACK_VERSION: "8.0.1"
-    - E2E_STACK_VERSION: "8.1.3"
-    - E2E_STACK_VERSION: "8.2.3"
-    - E2E_STACK_VERSION: "8.3.3"
-    - E2E_STACK_VERSION: "8.4.3"
-    - E2E_STACK_VERSION: "8.5.3"
-    - E2E_STACK_VERSION: "8.6.2"
-    - E2E_STACK_VERSION: "8.7.1"
-    - E2E_STACK_VERSION: "8.8.2"
-    - E2E_STACK_VERSION: "8.9.2"
-    - E2E_STACK_VERSION: "8.10.4"
-    - E2E_STACK_VERSION: "8.11.4"
-    - E2E_STACK_VERSION: "8.12.2"
     - E2E_STACK_VERSION: "8.13.0"
     # current stack version 8.14.x is tested in all other tests no need to test it again
     - E2E_STACK_VERSION: "8.15.0-SNAPSHOT"
```

Something is wrong here? The current version is `8.13.2` not a `8.14.x`.

```suggestion
    # current stack version 8.13.x is tested in all other tests no need to test it again
    - E2E_STACK_VERSION: "8.14.0-SNAPSHOT"
    - E2E_STACK_VERSION: "8.15.0-SNAPSHOT"
```